recipes/pygame/patches/0006-add-some-more-display-tags-to-tests-modules.patch

```diff
@@ -0,0 +1,1094 @@
+ 
+     def test_pixels_green(self):
++        """
++        |Tags:display|
++
++        """
+         self._test_pixels_rgb("green", 1)
+ 
+     def test_pixels_blue(self):
++        """
++        |Tags:display|
++
++        """
+         self._test_pixels_rgb("blue", 2)
+ 
+     def _test_pixels_rgb(self, operation, mask_posn):
+diff --git a/test/time_test.py b/test/time_test.py
+index 86554b9d..c2fc6763 100644
+--- a/test/time_test.py
++++ b/test/time_test.py
+@@ -240,6 +240,10 @@ class TimeModuleTest(unittest.TestCase):
+             self.assertAlmostEqual(ticks_diff, time_diff, delta=delta)
+ 
+     def test_set_timer(self):
++        """
++        |Tags:display|
++
++        """
+         """Tests time.set_timer()"""
+         """
+         Tests if a timer will post the correct amount of eventid events in
+diff --git a/test/touch_tags.py b/test/touch_tags.py
+index 0135fc49..5ab1645d 100644
+--- a/test/touch_tags.py
++++ b/test/touch_tags.py
+@@ -1 +1 @@
+-__tags__ = ["SDL1_ignore"]
++__tags__ = ["display", "SDL1_ignore"]
+diff --git a/test/transform_test.py b/test/transform_test.py
+index 65cfbc6c..a9ad0f72 100644
+--- a/test/transform_test.py
++++ b/test/transform_test.py
+@@ -761,8 +761,9 @@ class TransformModuleTest(unittest.TestCase):
+ 
+     def test_laplacian(self):
+         """
+-        """
++        |Tags:display|
+ 
++        """
+         SIZE = 32
+         s1 = pygame.Surface((SIZE, SIZE))
+         s2 = pygame.Surface((SIZE, SIZE))
+@@ -792,6 +793,8 @@ class TransformModuleTest(unittest.TestCase):
+ 
+     def test_laplacian__24_big_endian(self):
+         """
++        |Tags:display|
++
+         """
+         pygame.display.init()
+         try:
+@@ -883,6 +886,10 @@ class TransformModuleTest(unittest.TestCase):
+         self.assertEqual(sr.get_at((0, 0)), (10, 53, 50, 255))
+ 
+     def test_average_surfaces__24_big_endian(self):
++        """
++        |Tags:display|
++
++        """
+         pygame.display.init()
+         try:
+             surf_1 = pygame.image.load(
+@@ -1251,6 +1258,10 @@ class TransformDisplayModuleTest(unittest.TestCase):
+         pygame.display.quit()
+ 
+     def test_flip(self):
++        """
++        |Tags:display|
++
++        """
+         """ honors the set_color key on the returned surface from flip.
+         """
+         image_loaded = pygame.image.load(example_path("data/chimp.png"))
+@@ -1296,6 +1307,10 @@ class TransformDisplayModuleTest(unittest.TestCase):
+         self.assertEqual(surf.get_at((0, 0)), surf2.get_at((0, 0)))
+ 
+     def test_flip_alpha(self):
++        """
++        |Tags:display|
++
++        """
+         """ returns a surface with the same properties as the input.
+         """
+         image_loaded = pygame.image.load(example_path("data/chimp.png"))
+diff --git a/test/video_tags.py b/test/video_tags.py
+new file mode 100644
+index 00000000..93a1c469
+--- /dev/null
++++ b/test/video_tags.py
+@@ -0,0 +1 @@
++__tags__ = ["display"]
+-- 
+2.37.0.windows.1
```

Comment:
not sure why this whole patch needed, atleast this doesn't cause issues on the pygame side CI. As an above comment mentioned, are you setting `SDL_VIDEODRIVER` and `SDL_AUDIODRIVER` correctly before the tests?